### ISSUE TITLE
App doesn't refresh when source feature service is updated

### ISSUE THREAD

**Issue by kerryrobinson** [Esri/map-tour-storytelling-template-js#58]
We have a script that runs to recreate the data in the feature service that we are using in our Map Tour app (it deletes the original features and creates new ones). If we run the script, the app will not automatically update with the new features. We have to go back into the app in edit mode, where it says "One or multiple points added outside of the interactive builder are hidden.", and unhide the new features.

We are trying to use the Map Tour as part of an automated process on a daily basis, so we want to avoid manual intervention with the app after an update if possible.

Is this intended behavior of the app?  If features are just updated instead of deleted and created again, I assume this won't happen.

Is there any way to get around this where new features are automatically un-hidden in the app?

**Comment by glazou** [comment]
Yes this is the designed behavior. That behavior open interesting workflow to validate your data before they are published but we realize that this is not ideal in all cases and there should be an option to control the behavior.

There is two workaround that are not ideal:
- create a new MapTour by "sharing your webmap" and selecting the MapTour app. When the builder opens, don't click save the save button and don't use the builder at all. In that case because the app has never got confirmation from you of what points you want in your story, it's going to assume you want all of them: so any change to the Feature Service will be reflected in your story. The bad news is that you are not able to use the builder for anything so if you need to enable any options or customize the title/subtitle you can't do that. For the title/subtitle, you can edit the application item and those change will appear in the story (title and description of the item).
- use https://ago-assistant.esri.com to modify the story configuration to set the `order` property to `null`. This reset the order configuration and will put you in the same context than workaround above. You will have to do that after every use of the builder. That workaround present risk of breaking your application, it will not affect the data in your Feature Service but the MapTour could break.

![image](https://cloud.githubusercontent.com/assets/994078/17939160/0fbd6ef8-69de-11e6-9cd6-7ea7c92b206d.png)

**Comment by kerryrobinson** [comment]
OK, thanks for the answer and suggested workarounds.
